test/CheckoutSdkTest/Payments/Hosted/PaymentsClientTest.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Checkout.Common;
+using Moq;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Checkout.Payments.Hosted
+{
+    public class PaymentsClientTest : UnitTestFixture
+    {
+        private const string HostedPayments = "/hosted-payments";
+        private const string Reference = "ORD-1234";        
+
+        private readonly Mock<IApiClient> _apiClient = new Mock<IApiClient>();
+        private readonly Mock<CheckoutConfiguration> _configuration;
+        private readonly Mock<SdkCredentials> _sdkCredentials = new Mock<SdkCredentials>(PlatformType.Default);
+        private readonly SdkAuthorization _authorization = new SdkAuthorization(PlatformType.Default, ValidDefaultSk);
+        private readonly Mock<IHttpClientFactory> _httpClientFactory = new Mock<IHttpClientFactory>();
+        private HostedPaymentResponse _hostedPaymentResponse = new HostedPaymentResponse();
+
```

Comment:
Extra blank like here ^^

Reply:
Should I add an extra line or remove that extra blank line?